src/common/exif.cc

```diff
@@ -1254,9 +1260,14 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
             mat3mul(img->d65_color_matrix, correctmat[5], colmatrix[sel_illu]);
             break;
           case DT_LS_CoolWhiteFluorescent:
+          case DT_LS_Fluorescent:
```

Comment:
CoolWhiteFluorescent is assumed to be F2 illuminant (judging by the matrix and actual description of F2 matches) and it has temperature of 4230 K. Fluorescent is cited to have temperature of 3800 K by Lumariver manual. It's not really close to each other...

Reply:
Again, DNG SDK.

Reply:
DNG SDK is somewhat weird in this sense (e.g. why CoolWhiteFluorescent and not WhiteFluorescent), but OK, makes sense. BTW, we're missing other fluorescent illuminants (WarmWhite, etc.).

Reply:
Actually, they define CoolWhiteFluorescent as F9 and Fluorescent as F2, so they should indeed not be aliased here...

Reply:
As for the others, I could calculate and add them the same way as I did for 3200K - even easier as xy coords are defined already...

Reply:
I think maybe we're looking at different parts of DNG SDK after all, here's what I found: https://github.com/aizvorski/dng_sdk/blob/master/source/dng_camera_profile.cpp#L135

Reply:
Yes, but also take a look at `dng_xy_coord.cpp` of a more recent package, that one is too old.

Reply:
Indeed, thanx for that. IMHO it's logical to use then that as a reference.

Reply:
Updated now against that more up-to-date reference.

Reply:
Still missing the adaptation for the few remaining fluorescents though...